src/tribler-core/tribler_core/components/tag/db/tag_db.py

```diff
@@ -0,0 +1,164 @@
+import datetime
+import logging
+from enum import IntEnum
+from typing import List, Optional
+
+from pony import orm
+
+from tribler_core.utilities.path_util import Path
+
+
+class Operation(IntEnum):
+    ADD = 1
+    REMOVE = 2
+
+
+class TagDatabase:
```

Comment:
@devos50 @kozlovsky 

The DB has been added. Feel free to review it.

If you see some misspellings or incorrectly worded sentences in the comments, please write about it.

Reply:
Tomorrow I'll add limits for the tags and tags operations. 
Still not sure how it should be implemented.

Reply:
@drew2a to implement the constraint on the maximum number of tags operations, you can add an attribute `TorrentTag.operations_count = Required(int, default=0)`. You can modify it incrementally from the Python code when adding new operations. Or you can calculate operations count on the fly and check the value before adding a new operation to the database: `torrent_tag.operations.count()`. It is simpler but a bit less efficient.

The same is for the maximum number of tags for the torrent: you can add an attribute `Torrent.tags_count = Required(int, default=0)`, or just check `torrent.tags.count()` when necessary.